### ISSUE TITLE
SkiaSharp XF Sample in .NETStandard 1.4

### ISSUE THREAD

**Issue by JimSEOW** [mono/SkiaSharp#352]
I am testing bringing the XF Skiasharp sample to .NETStandard 1.4 for UWP
The only issue is instead of PCLStorage v1.0.2, I choose [PCLStorage.Standard compile with .NETStandard 1.4] (https://github.com/losttech/PCLStorage.Desktop/tree/netStandard)

Two samples fail
- CreatePdfSample
- CreateXpsSample
 because of dependency on FileSystem.cs  which is missing in PCLStorage.Standard branch

**Comment by mattleibow** [comment]
@JimSEOW Thanks for looking into this. With .NET Standard 1.3+, there is no need for the PCLStorage as the File APIs are all available.

If we are going .NET Standard, rather switch to that API.

**Comment by JimSEOW** [comment]
@mattleibow  UWP6.0 is expected to [be released end of this month](https://github.com/dotnet/corefx/milestones). By then, we could bring Xamarin Foms to .NET standard 2.0 for iOS, Android and UWP and then we could re-visit this topics.

**Comment by mattleibow** [comment]
I had a look, and managed to get the sample to .NET Standard: 37fae29dfc165100366b97877edad7fc0f5e5e65

Thanks for the issue and the research. It is still a bit rough, (stable) Xamarin.Forms does not yet nicely support .NET Standard, so I had to add: `<PackageTargetFallback>portable-net45+win8+wpa81+wp8</PackageTargetFallback>` to the .csproj. Not too bad, but hopefully as soon as 2.4 goes stable, we can remove it.

Also, UWP is still a bit annoying, it doesn't support a pure .csproj format. I moved the package references into the project file, but I still have to keep that project.json around. All the other projects use the new packages format.
